specifications/api/swagger.yaml

```diff
@@ -4792,3 +5105,72 @@ paths:
           required: false
           description: check this url for health instead of the currently configured one
       responses: *responsesGetSmashHealth
+
+  /shared-wallets:
+    post:
+      operationId: postSharedWallet
+      tags: ["Shared Wallets"]
+      summary: Create
+      description: |
+        <p align="right">status: <strong>unstable</strong></p>
```

Comment:
Just before we merge this, is it worth clarifying the shared-wallet API docs.
The feature is not yet fully implemented - so perhaps add words to that effect?